packages/pn-pa-webapp/src/pages/NewApiKey.page.tsx

```diff
@@ -141,6 +141,8 @@ const NewApiKey = () => {
                           value={formik.values.groups}
                           options={groups}
                           id="groups"
+                          openText=""
```

Comment:
Do the updates on PnAutocomplete in pn-commons, so all instances of autocomplete will be fixed